### ISSUE TITLE
Request: Vengeance Ring mob blacklist

### ISSUE THREAD

**Issue by fl0ppat** [rubensworks/EvilCraft#501]
Can you add black list for mob spawn, when Player use VengeanceRing?
Please :^)

**Comment by rubensworks** [comment]
There is an `entityBlacklist` config option for vengeance spirits, is this sufficient for you?

**Comment by fl0ppat** [comment]
Yup. This list balance gameplay, because Ring spawn random mobs (wither and other bosses too). 
I think some spirits should spawn only after kill.
